python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -510,7 +510,10 @@ def _setImageColormap(self, cmap):
 
         """
         if not isinstance(cmap, mpColors.Colormap):
-            cmap = getattr(matplotlib.cm, cmap)
+            try:
+                cmap = matplotlib.colormaps[cmap]
+            except AttributeError:
+                cmap = getattr(matplotlib.cm, cmap)
```

Comment:
I don't think we're trying to support matplotlib < 3.8 anymore, so we can drop the `try/except` here.